labs/ai-vision/analyze_video_workshop/notebooks/Offline_Video_Service.ipynb

```diff
@@ -199,16 +206,42 @@
     "<details>\n",
     "<summary><font size=\"3\">compartment_id</font></summary>\n",
     "    <font size=\"2\">The user should provide the compartment OCID to call the API.</font><br>\n",
-    "</details>\n",
+    "</details>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "243e9495",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "input_namespace = \"<INPUT_NAMESPACE>\"\n",
+    "input_bucket = \"<INPUT_BUCKET>\"\n",
+    "input_filename = \"<INPUT_FILENAME>\"\n",
+    "\n",
+    "output_namespace = \"<OUTPUT_NAMESPACE>\"\n",
+    "output_bucket = \"<OUTPUT_BUCKET>\"\n",
+    "output_prefix = \"<OUTPUT_PREFIX>\"\n",
+    "\n",
+    "compartment_id = \"<COMPARTMENT_ID>\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dce24213",
+   "metadata": {},
+   "source": [
+    "#### Set optional input variables\n",
     "\n",
     "<details>\n",
     "<summary><font size=\"3\">max_results</font></summary>\n",
-    "    <font size=\"2\">Provide the maximum number of results needed for image classification. This is an upper limit over the output classes, the API may detect lesser classes according to the image.</font><br>\n",
+    "    <font size=\"2\">Provide the maximum number of results needed for image classification. This is an upper limit over the output classes, the API may detect lesser classes according to the image. In case Object and Label Detection default value is 5. Whereas for Face Detection default value is 50 </font><br>\n",
```

Comment:
should be:
In case of Object and Label Detection

Reply:
Isn't it the same as you mentioned. I understand I have to change image classification to label detection in the first sentence

Reply:
"of" is missing